src/components/usa-skipnav/usa-skipnav.stories.js

```diff
@@ -0,0 +1,23 @@
+import "../../stylesheets/uswds.scss";
+import Component from "./usa-skipnav.twig";
+import Data from "./usa-skipnav.json";
+
+export default {
+  title: "Components/Skipnav",
+  argTypes: {
+    modifier: {
```

Comment:
Are there modifiers for this component? I didn't see any so just wanted to confirm

Reply:
There aren't modifiers. We should remove.